data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			`	"That sounds like a good idea," the captain says before walking off into the spaceport.`
+			`	You continue to question more captains in the spaceport over the course of the day with Xilin and Sayari. Some already knew of the Hai before, but many are new arrivals who heard of the way to the Hai through whispers in spaceport bars, or were sold coordinates by knowledgeable captains. Everyone seems peaceful enough in interacting with the Hai, but clearly the word is getting out more as time goes on.`
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		conversation
+			`You return to the briefing room at the secure port with Xilin and Sayari, and Xilin begins frantically copying notes into a terminal. He then pauses while staring at something on the screen.`
+			`	"I'm running each of the pilot's licenses and ship registrations through my copy of the Republic's pilot database. They've all come up as valid and with mostly clean records. All except for one." He turns the terminal screen toward you and Sayari, displaying an inputted license ID that the system says does not exist.`
+			`	"This is from that Berserker pilot. What's more is that if I run the ID of the ship, it's registered as stolen."`
+			`	Sayari gives a look of concern. "Are you suggesting that this captain was a criminal?"`
+			`	"Highly likely," Xilin responds. "A pirate with a stolen ship and a counterfeit license."`
+			`	"I will inform our spaceport authorities to look out for this captain and his ship," Sayari says. "It may also be beneficial if you were to make this database available to us, so that we may catch other potential criminal humans."`
+			`	"Unfortunately, I'd have to gain authorization for that," says Xilin. "But I'll see what I can do."`
+			choice
+				`	"Should I help track down that captain?"`
+					goto track
+				`	"So are pirates trying to sneak their way into Hai territory?"`
+			`	"That may be the case," Xilin says. "We'd be stupid to suspect this as only having been an isolated incident."`
+				goto pirates
+			label track
+			`	"No, leave that to the Hai," Xilin says. "At least for so long as this is our only lead on pirate activity in Hai territory, although I imagine that we'll find more if we go looking for it."`
+			label pirates
+			`	"We have had run-ins with your pirates before," Sayari says. "Most of the time they cause only mischief. Very rarely do that raid our fleets and cause large trouble. I fear though that this situation will only embolden them."`
+			`	Just then, Alondo poke his head into the room looking somewhat exasperated. "Captain <last>. Could you give Remington a hand? He's out on the landing pads." He poke his head back out of the room, and you hear his heavy footsteps grow quieter as he walks away.`
+			`	"You go help Remington," Xilin says. "I'll continue my work here."`
```

Comment:
This transition could be a little less abrupt.

Reply:
Do you have any suggestion for how to change it?

Looking for the FW campaign, conversations ending like this, with a line from some character, isn't too out of the ordinary. It's been said before that this is a game, not a novel, and in that sense I don't think we necessarily need to have some sort of transition out of every conversation. When I was writing these, I certainly couldn't think of something that wouldn't just be a repetition of something I'd said before. (In a similar vein, I think that the very first line of conversations/mission chains can be a pain to write, because it's almost always some variation of "you land" or "you're in the spaceport when..." so long as we hold that we need some transition into the conversation instead of being able to be somewhat abrupt.)